sys/page_revisions.js

```diff
@@ -679,10 +681,18 @@ PRS.prototype.getRevision = function(hyper, req) {
             }
         });
     }
+
+    function fetchFromMW() {
```

Comment:
The naming suggests that this is only fetching, but it is really saving as well. Perhaps something like fetchStoreAndCheckMWRevision could be better? Or, actually, is there a call to fetchAndStoreMWRevision that doesn't want the checking part?